.github/workflows/ci.yaml

```diff
@@ -14,29 +14,29 @@ env:
   BATCH_COUNT: 6
```

Comment:
We should probably split up the batch count too. The examples will likely go a lot faster, so we don't need nearly as many batches.